ApplicationPattern.yaml

```diff
@@ -589,6 +589,18 @@ paths:
                         'If provided, own TCP port, which has to be documented (but not be directly used for formulating the callback) into the TcpServer with
                         find [/core-model-1-4:control-construct/logical-termination-point=*-1-0-0-tcp-s-*/layer-protocol=0/tcp-server-interface-1-0:tcp-server-interface-pac/tcp-server-interface-configuration/local-protocol] == https
                         update [/core-model-1-4:control-construct/logical-termination-point=*-1-0-0-tcp-s-*/layer-protocol=0/tcp-server-interface-1-0:tcp-server-interface-pac/tcp-server-interface-configuration/local-port]'
+                    preceding-application-name:
+                      type: string
+                      description: >
+                        'If provided, the new application will address /v1/bequeath-your-data-and-die at the preceding application after receiving /v1/embed-yourself
```

Comment:
Shall we also make this input to update [/core-model-1-4:control-construct/logical-termination-point=\*-http-c-or-\*/layer-protocol=0/http-client-interface-1-0:http-client-interface-pac/http-client-interface-configuration/application-name]'.
Reason for this ask is as follow , 
1. As per the description , "/v1/bequeath-your-data-and-die will not be sent, if application-name in *-http-c-or-* == OldRelease" , if application-name is OldRelease , then /v1/bequeath-your-data-and-die will not be initiated.
2. As a part of this API request , if we are not updating the application-name of \*-http-c-or-\* , then , there is no consequent service that updates this information.
Note : Embed-yourself will not provide the oldRelease application-name , it will update only the tcp information of the oldRelease.

Reply:
> Shall we also make this input to update [/core-model-1-4:control-construct/logical-termination-point=*-http-c-or-*/layer-protocol=0/http-client-interface-1-0:http-client-interface-pac/http-client-interface-configuration/application-name]'

Yes, we must.
Many thanks for indicating.
Beware of changed semantics in the new proposal.